editor/components/webgl/surface.ts

```diff
@@ -45,20 +45,20 @@ export class WebGLSurface implements AfterViewInit, OnDestroy {
     private gl_not_supported: boolean = false;
     private tex_loader: TextureLoader;
     private _loop: () => void;
-    private mouse_handler: MouseHandler;
-    private key_handler: KeyHandler;
-    private pipeline: Pipeline = undefined;
+    private mouse_handler?: MouseHandler;
```

Comment:
It's a shortcut for `MouseHandler | undefined` and also the recommended annotation for optional properties.